content/python/concepts/built-in-functions/terms/super/super.md

```diff
@@ -0,0 +1,74 @@
+---
+Title: 'super()'
+Description: 'Returns a temporary object that allows a given class to inherit the methods and properties of a parent or sibling class.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Objects'
+  - 'Inheritance'
+  - 'Classes'
+  - 'Functions'
+  - 'Inheritance'
+  - 'Methods'
+  - 'Objects'
+  - 'OOP'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`super()`** function returns a temporary object that allows a given [class](https://www.codecademy.com/resources/docs/python/classes) to [inherit](https://www.codecademy.com/resources/docs/python/inheritance) the methods and properties of a parent or sibling class.
+
+## Syntax
+
+```pseudo
+super().method()
+super(type, object)
+```
+
+`.method()` is the accessed method of the parent class. `type` specifies the type object of the parent class and object is an instance or subtype of the `type`.
+
```

Comment:
Can we place a second \```pseudo snippet here that looks like this:

```pseudo
class A():
  method(arg):
    # Method code starts here

class B(A):
  method(arg):
    super().method(arg)
    # Method code starts here
```

Reply:
Updated.